src/clue.js

```diff
@@ -3,24 +3,207 @@
 // Suspects Array
 
 const suspectsArray = [];
+let suspect1 = {
+    firstName: "Jon",
+    lastName: "Snow",
+    occupation: "Nigth's Watch",
+    age: 28,
+    description: "Doesn't want to be king",
+    image: "Dark Hair",
+    color: "Caucasian"
+}
+
+let suspect2 = {
+    firstName: "Cersei",
+    lastName: "Lannister",
+    occupation: "Queen Mother",
+    age: 39,
+    description: "Want's to be queen",
+    image: "Blonde hair",
+    color: "Caucasian"
+}
+
+let suspect3 = {
+    firstName: "Missandei",
+    lastName: "of Naath",
+    occupation: "Councelor",
+    age: 27,
+    description: "Want's to serve Daenerys",
+    image: "Dark Hair",
+    color: "Brown skin"
+}
+
+let suspect4 = {
+    firstName: "Jorah",
+    lastName: "Mormont",
+    occupation: "Knigth",
+    age: 46,
+    description: "Want's to be pardoned",
+    image: "Blonde hair",
+    color: "Caucasian"
+}
+
+let suspect5 = {
+    firstName: "Arya",
+    lastName: "Stark",
+    occupation: "Blade Dancer",
+    age: 18,
+    description: "Want's to kill Cersei",
+    image: "Dark hair",
+    color: "Caucasian"
+}
+
+let suspect6 = {
+    firstName: "Theon",
+    lastName: "Greyjoy",
+    occupation: "Traveler",
+    age: 46,
+    description: "Want's to get back the iron Islands",
+    image: "Dark hair",
+    color: "Caucasian"
+}
+
+suspectsArray.push(suspect1, suspect2, suspect3, suspect4, suspect5, suspect6);
```

Comment:
There is no need to declare the objects one-by-one and push them inside the array, you can declare the objects inside the array straight away!

```
const suspectsArray = [
    {firstName: "Just", lastName:"Snow", age: 28, color:"red"},
    {firstName: "An", lastName:"Snow", age: 30, color:"red"},
    {firstName: "Example", lastName:"Snow", age: 23, color:"red"}
];
```
Same for roomsArray and weaponsArray